cmd/libsnap-confine-private/classic.c

```diff
@@ -56,3 +57,18 @@ sc_distro sc_classify_distro(void)
 		return SC_DISTRO_CLASSIC;
 	}
 }
+
+bool sc_is_debian_like(void)
+{
+	FILE *f SC_CLEANUP(sc_cleanup_file) = fopen(os_release, "r");
+	if (f == NULL) {
+		return false;
+	}
+	char *id_like;
```

Comment:
otherwise we're leaking id_like

```suggestion
	char *id_like SC_CLEANUP(sc_cleanup_string) = NULL;
```

Reply:
Thanks, updated

Reply:
pushed this along with a unit test